.template/ios/fastlane/Constants/Constants.rb

```diff
@@ -0,0 +1,87 @@
+# frozen_string_literal: true
+
+class Constants
+  #################
+  #### PROJECT ####
+  #################
+
+  # Workspace path
+  def self.WORKSPACE_PATH
+    './Runner.xcworkspace'
+  end
+
+  # Project path
+  def self.PROJECT_PATH
+    './Runner.xcodeproj'
+  end
+
+  # bundle ID for Staging app
+  def self.BUNDLE_ID_STAGING
+    'co.nimblehq.flutter.template.staging'
```

Comment:
I think we should create a placeholder here and replace it when running `setup.py` script 🤔

Reply:
@markgravity yes, I will recheck the init script and create a task if needed. Thanks for reminding 😉